### ISSUE TITLE
Documentation sort or sort_by

### ISSUE THREAD

**Issue by brkeh** [jqlang/jq#2103]
https://github.com/stedolan/jq/blob/ccc79e592cfe1172db5f2def5a24c2f7cfd418bf/docs/content/manual/manual.yml#L1317

Should this be
`sort_by` may be used to sort by a particular field ?

If it's possible to do this with just sort, can somebody give an example?

Thanks very much

**Comment by itchyny** [comment]
- The document was firstly written at https://github.com/stedolan/jq/commit/11965aaa2e9cf2b09c431e682cdc459b99de60fb#diff-d2fa3f3320a35df5eca0fe665267bd2fR509, where it says `sort_by` may be used to sort...
- The document was changed to `sort` at https://github.com/stedolan/jq/pull/426/files#diff-d2fa3f3320a35df5eca0fe665267bd2fR913. This pull request implements `sort(f)` and suggests the deprecation of `sort_by(f)`.
- Then `sort(f)` was removed at https://github.com/stedolan/jq/commit/a0a350f46808229bb25fe16ca8e82e4c2fe76d7c.
- And the document was updated at https://github.com/stedolan/jq/commit/07d8d5c5ef08b942b4cd0f8b9d1af6beb870841f#diff-d2fa3f3320a35df5eca0fe665267bd2fR1009, which undeprecates `sort_by(f)` but lefts the sentence in this issue.

In summary, the sentence is surely all about the `sort_by` function, and it make sense to restore the original paragraph.

**Comment by brkeh** [comment]
Well done @itchyny. That's great analysis, thanks.
